### PR TITLE
Update to JupyterHub 0.9.2

### PR DESCRIPTION
**Pull request by manics**
Updates to the latest packages, so this can be used for setting up an alternative fallback server for JupyterHub

Tag: `3.0.0`

See https://github.com/openmicroscopy/prod-playbooks/pull/93

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +18,22 @@ For instance, you can use the `openmicroscopy.docker` role.
...
+- `idr_jupyter_proxy_token`: Persistent token to allow connections to persist over a restart
+- `idr_jupyter_cull_options`: List of parameters to be passed to the cull idle servers service, see https://github.com/jupyterhub/jupyterhub/blob/0.9.2/examples/cull-idle/cull_idle_servers.py
+- `idr_jupyter_notebook_remove_containers`: Automatically stop and delete containers when the hub is restarted, default `True`
+- `idr_jupyter_notebook_system_uid`: UID of the notebook Docker container, default `1000`
 - `idr_jupyter_notebook_volumes`: JSON mapping of host volumes to internal notebook docker paths
-- `idr_jupyter_users`: List of users that can use Jupyterhub
-- `idr_jupyter_admins`: List of users with Jupyterhub admin privileges
-- `idr_jupyter_urlbase`: The base for URLs created by Jupyterhub
+- `idr_jupyter_users`: List of users that can use Jupyterhub, if empty all users will be allowed, default `[root]`
+- `idr_jupyter_admins`: List of users with Jupyterhub admin privileges, default `[root]`
```
Comment (jburel) [inline README.md]: JupyterHub

File: tasks/main.yml [hunk 2]
```diff
@@ -48,32 +40,27 @@
...
-    virtualenv_command: "{{ idr_jupyter_basedir }}/virtualenv-15.1.0/virtualenv.py"
-    virtualenv_python: python3
+    virtualenv_command: /usr/bin/python3.6 -m venv
+  notify:
+  - restart jupyterhub
+
+- name: jupyterhub | download cull idle script
+  become: yes
+  get_url:
+    checksum: "sha256:87e780d986f2f4a896f90c18b82806d364a926985ff549e88208dce6a87a205a"
```
Comment (jburel) [inline tasks/main.yml:62]: maybe add a note to keep version and checksum in synch

**Comment by manics** [comment]
@jburel updated in response to your comments

**Comment by jburel** [comment]
now that the roles are used in larger context that IDR 
we might confuse people
the delineation between idr and openmicroscopy is tricky
This is outside the scope of this PR
Things look good

**Pull request merged**